portfolio/src/main/webapp/script.js

```diff
@@ -85,4 +130,13 @@ function padZero(text) {
         text = "0" + text;
     }
     return text;
+}
+
+// make buttons with href onclick
+function buttonHref(text, link) {
```

Comment:
A good practice is to always use a verb to start a function, for example, you can call it createButtonWithHref

Reply:
done, thanks!